components/menu/menu-item-mixin.js

```diff
@@ -40,7 +40,7 @@ export const MenuItemMixin = superclass => class extends superclass {
 			 */
 			text: { type: String },
 			/**
-			 * Provide a description for the menu item that will be used by screen readers
+			 * ACCESSIBILITY: Provide a description for the menu item that will be used by screen readers
```

Comment:
```suggestion
			 * ACCESSIBILITY: A description of the menu item that will be used by screen readers for additional context
```

Reply:
Related to this change - we're not entirely consistent on "screen readers" vs "screenreaders". Which should we use?

Reply:
My opinion would be "screen reader" based on google